clean_project/clean_file.py

```diff
@@ -0,0 +1,17 @@
+import string
+
+
+def clean_up(src_path, dest_path):
+    with open(src_path) as source_file:
+        with open(dest_path, "w") as dest_file:
+            for line in source_file:
+                table = string.maketrans(line, line)
+                new_line = line.translate(table, string.punctuation)
+                dest_file.write(new_line)
+            source_file.close()
```

Comment:
you don't need to close the files: this is the whole point of opening them in the with statement, as the files will be closed automatically after the block or in case of abnormal termination